### DISCUSSION TITLE
how to hide the bar

### DISCUSSION THREAD

**Discussion by woquchonglang** [ashish0kumar/windots#16] [Issues & Help] [answered]
hey, I am using your dotfiles, to be honestly, you dotfiles are so beatiful, but I want to ask how to hide the bar like the image?
![image](https://github.com/user-attachments/assets/577f399b-4aca-41e7-b478-89aa39238c0b)

**Comment by ashish0kumar** [comment, marked as answer]
If you are using [Windows Terminal](https://github.com/microsoft/terminal) then, 
Open settings > startup > launch parameters and set the launch mode to focus

![Screenshot 2024-10-26 215317](https://github.com/user-attachments/assets/88fd4d6d-ab7e-4ce4-b028-171aa157efaf)
